karabo_bridge/simulation.py

```diff
@@ -93,13 +93,21 @@ def corr_passport(self):
             '%s/CAL/RELGAIN_CORR_Q1M1' % domain
         ]
         return passport
+    @property
+    def _img_shape(self):
```

Comment:
I'm confused. There is already a `data_shape` property. Can you use it instead of making a new one?

Reply:
Yes, but the shape of imge.data is different if we have corrected data. Hence I added this property the is used only to get the right shape when creating image data.